public/API/API_GetGame.php

```diff
@@ -22,29 +22,31 @@
  *  string     Released                   release date information for the game
  */
 
+use App\Platform\Models\Game;
+
 $gameID = (int) request()->query('i');
 
-$game = getGameData($gameID);
+$game = Game::with('system')->find($gameID);
 
-if ($game === null) {
+if (!$game) {
     return response()->json();
+} else {
```

Comment:
nit: This `else` is redundant.

Reply:
Without this else, the endpoint returns 1 instead of an empty json response

Reply:
I'm experiencing something different:

```php
if (!$game) {
    return response()->json();
}

return response()->json([
    'Title' => $game->Title,
    'GameTitle' => $game->Title,
    'ConsoleID' => $game->ConsoleID,
    'ConsoleName' => $game->system->Name,
    'Console' => $game->system->Name,
    'ForumTopicID' => $game->ForumTopicID,
    'Flags' => 0, // Always '0'
    'GameIcon' => $game->ImageIcon,
    'ImageIcon' => $game->ImageIcon,
    'ImageTitle' => $game->ImageTitle,
    'ImageIngame' => $game->ImageIngame,
    'ImageBoxArt' => $game->ImageBoxArt,
    'Publisher' => $game->Publisher,
    'Developer' => $game->Developer,
    'Genre' => $game->Genre,
    'Released' => $game->Released,
]);
```

```
GET http://localhost:64000/API/API_GetGame.php?z=...&y=...&i=9999999
200

[]
```

Reply:
Ah I see what I did wrong, I removed the whole empty return.